File: accounts/accounts/report/profit_and_loss_statement/profit_and_loss_statement.js
// Copyright (c) 2016, Frappe and contributors
// For license information, please see license.txt
/* eslint-disable */

frappe.query_reports["Profit and Loss Statement"] = {
	"filters": [
		{
			"fieldname":"from_fiscal_year",
			"label": __("Start date"),
			"fieldtype": "Link",
			"options": "Fiscal Year",
			"reqd": 1
		},
		{
			"fieldname": "to_fiscal_year",
			"label": __("End date"),
			"fieldtype": "Link",
			"options": "Fiscal Year",
			"reqd": 1
		}

	]
};

File: accounts/accounts/report/profit_and_loss_statement/profit_and_loss_statement.py
# Copyright (c) 2013, Frappe and contributors
# For license information, please see license.txt

from __future__ import unicode_literals
# import frappe

def execute(filters=None):
	if not filters:
		return [], []
	period_list = get_period_list(filters.from_fiscal_year, filters.to_fiscal_year)

def get_period_list(from_fiscal_year, to_fiscal_year):
	
